### DISCUSSION TITLE
Question: expand code snippet in readme to full height?

### DISCUSSION THREAD

**Discussion by linrongbin16** [community/community#69524] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

when using permanent link in readme markdown, it will expand to a code snippet, but is the height/width configurable?

I want to set my code snippet to full height/width, how should I do?

**Comment by krushnarout** [comment, marked as answer]
Markdown doesn't have specific mechanisms to directly control the height and width of code blocks or code snippets.

If the environment where you're rendering the Markdown supports HTML and CSS, you can use HTML within your Markdown to style the code block. For example:

```html
<div style="width: 100%; height: 100%">
    <pre>
        <code>
            Your code goes here...
        </code>
    </pre>
</div>
```
